scripts/govDocES.py

```diff
@@ -0,0 +1,51 @@
+import os
+from elasticsearch.exceptions import NotFoundError
+
+from model import Work, Edition, ESWork
+from main import loadEnvFile
+from managers import DBManager, ElasticsearchManager
+
+
+def main():
+
+    loadEnvFile('local-qa', fileString='config/{}.yaml')
+
+    dbManager = DBManager(
+        user= os.environ.get('POSTGRES_USER', None),
+        pswd= os.environ.get('POSTGRES_PSWD', None),
+        host= os.environ.get('POSTGRES_HOST', None),
+        port= os.environ.get('POSTGRES_PORT', None),
+        db= os.environ.get('POSTGRES_NAME', None)
+    )
+
+    esManager = ElasticsearchManager()
+    esManager.createElasticConnection()
+
+    dbManager.generateEngine()
+
+    dbManager.createSession()
+
+    for work in dbManager.session.query(Work) \
+        .join(Edition) \
+        .filter(Edition.measurements != None) \
+        .filter(Edition.measurements != []) \
+        .filter(Edition.measurements != [{}]).all():
+            for dict in work.edit.measurements:
+                if dict['type'] == "government_document":
+                    if dict['value'] == "1":
+                        try:
+                            workRec = ESWork.get(work.uuid, index=esManager.index)
+                            workRec.is_government_document = True
+                            workRec.save()
```

Comment:
for speed/performance I think you can add a `break` here so you're not saving the same work multiple times in case it has multiple editions with that measurement.